whosonfirst/view/shapefile.js

```diff
@@ -23,13 +23,24 @@ module.exports = (feat, params) => {
   // exportable filters
   if (!filters.exportable(feat, spr)) return null
 
+  // logic for placetype local property
+  if( _.first(feature.getPlacetypeLocal(feat))) {
+    if( _.first(feature.getPlacetypeLocalLatin(feat))):
+      placetype_local = concat( _.first(feature.getPlacetypeLocal(feat)), " (", _.first(feature.getPlacetypeLocalLatin(feat), ")";
```

Comment:
This parenthesis field feels a little uncomfortable to me. I guess it's because the string value is a concatenation of two fields with some punctuation added.

An alternative option would be to introduce a new field, this would allow each consumer to mix and match these fields and format them as they desired for their individual needs.

It's not a big deal when used as a label, but it would make the locallatin information difficult to recover from the string if someone wanted to use it a different way.

Reply:
To me the SHP is a "non-expert" distribution of WOF so it's OK to transform it a bit more during ETL to widen the audience to local and non-local users. 

The DBF is also hard limited at 2GB, and null values add something to file size in this format, so less columns are better, and there's far less than 1% fill rate on the latin labels.